documentation/modules/security/proc-configuring-external-clients-to-trust-cluster-ca.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='configuring-external-clients-to-trust-cluster-ca-{context}']
+= Configuring external clients to trust the cluster CA
+
+This procedure describes how to configure a Kafka client that resides outside the Kubernetes cluster – connecting to the `external` listener on port 9094 – to trust the cluster CA certificate.
+Follow this procedure when setting up the client and during the renewal period, when the old clients CA certificate is replaced.
+
+Follow the steps to configure trust certificates signed by the cluster CA for Java-based Kafka Producer, Consumer, and Streams APIs.
+
+Choose the steps according to the certificate format for the cluster CA: PKCS#12 (`.p12`) or PEM (`.crt`).
+
+IMPORTANT: The `_<cluster-name>_-cluster-ca-cert` `Secret` will contain more than one CA certificate during CA certificate renewal.
```

Comment:
Is "during CA certificate renewal" concrete enough?

"During the CA certificate renewal period"?